front/src/applications/operationalStudies/__tests__/sampleData.ts

```diff
@@ -351,8 +352,8 @@ export const electrificationRangesLarge: ElectrificationRange[] = [
   },
 ];
 
-export const trainScheduleTooFast: TrainScheduleResult = {
-  id: 98,
+export const trainScheduleTooFast: TrainScheduleResultWithTrainId = {
+  id: 'train-98' as TrainScheduleId,
```

Comment:
It should be trainschedule now I think
```suggestion
  id: 'trainschedule-98' as TrainScheduleId,
```

Reply:
Good catch !